src/AppInstallerCommonCore/MSStoreDownload.cpp

```diff
@@ -910,6 +911,12 @@ namespace AppInstaller::MSStore
             {
                 SFS::RequestParams sfsClientRequest;
                 sfsClientRequest.productRequests = { {std::string{ wuCategoryId }, {}} };
+                if (AppInstaller::Settings::Network().GetProxyUri())

```

Comment:
why not get the optional a single time only?
```
const auto& proxyUri = AppInstaller::Settings::Network().GetProxyUri();
if (proxyUri)
{
    sfsClientRequest.proxy = *proxyUri;
}
```

Reply:
Both fixed, thanks Arthur